web/src/main/java/org/cbioportal/web/error/GlobalExceptionHandler.java

```diff
@@ -52,12 +60,13 @@
     @ExceptionHandler(SampleNotFoundException.class)
     public ResponseEntity<ErrorResponse> handleSampleNotFound(SampleNotFoundException ex) {
 
-        return new ResponseEntity<>(new ErrorResponse("Sample not found: " + ex.getSampleId()),
-                HttpStatus.NOT_FOUND);
+        return new ResponseEntity<>(new ErrorResponse("Sample not found in study " + ex.getStudyId() + ": " +
```

Comment:
It seems like we should document this as a API design standard ... calling a fetch endpoint and failing to match (some? all?) in the database produces a NOT_FOUND status code.